app/modules/ads/index.js

```diff
@@ -154,11 +187,6 @@ class Ads {
   startAdEngine() {
     const { AdEngine } = window.Wikia.adEngine;
 
-    if (!this.isAdStackEnabled()) {
```

Comment:
Why is this removed? Smart banner relies on `fanTakeoverResolver` decision. Does it work properly now?

Reply:
:facepalm:

Does it mean that smart banner is broken for some time already? It seems that fanTakeoverResolved isn't resolved at all when noads/ad stack is disabled :(

Reply:
I fixed smart banner by resolving the promise if ad stack is disabled.

Reply:
Thank you